docs/requirements.txt

```diff
@@ -3,3 +3,5 @@
 # SPDX-License-Identifier: Unlicense
 
 sphinx>=4.0.0
+adafruit_circuitpython_typing>=1.8.2
+typing_extensions>=4.4.0
```

Comment:
I don't think we need anything imported from `typing_extensions` so this can be removed.  Let me know if I'm forgetting or not seeing something though.

Reply:
I just checked, and it's automatically installed with `adafruit_circuitpython_typing` so I've deleted it. I've also moved from `docs/requirements.txt` to `/requirements.txt`, where it belonged in the first place . Also changed the underscores to dashes.  <slaps forehead emoji>

Reply:
Ah, thanks and good catches, I missed those other points!